### PR TITLE
add plots for color testing and use as examples

### PR DESCRIPTION
**Pull request by achafetz**
example plots to test out colors on

### CODE REVIEW

File: data-raw/example_plots.R [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+
+
+# LOLLIPOP CHART ----------------------------------------------------------
+
+
+  df_lollipop <- df_bar %>%
+    mutate(achievement = cumulative / targets)
+
+  df_lollipop %>%
```
Comment (tessam30) [inline data-raw/example_plots.R:117]: would it make more sense to use tidytext::reorder_within() to have the partners ordered within the Agencies?

`df_lollipop %>%
    mutate(partner_order = tidytext::reorder_within(primepartner, achievement, agency)) %>%   
    ggplot(aes(partner_order, achievement)) + 
    geom_segment(aes(x = partner_order, y = 0, xend = partner_order, yend = achievement), size = .9) +  
    geom_point(size = 3) +
    tidytext::scale_x_reordered() +
  ...`

File: data-raw/example_plots.R [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    tibble(fundingagency = sample(c("USAID", "CDC"), nrow(df_bar), replace = TRUE))
+
+  df_bar <- df_bar %>%
+    bind_cols(agency, .)
+
+  df_bar %>%
+    ggplot(aes(fct_reorder(primepartner, targets_q1, sum), cumulative)) +
+    geom_col(aes(y = targets_q1), fill = "gray90") +
+    geom_col() +
+    geom_errorbar(aes(ymin = targets_q1, ymax = targets_q1), color = "gray90", size = .9) +
```
Comment (tessam30) [inline data-raw/example_plots.R:99]: Wasn't sure what the purpose of this line was. Here is the plot result I'm getting w/ it.
[Plot result](https://github.com/USAID-OHA-SI/glitr/issues/14#issue-633632977)

File: data-raw/example_plots.R (hunk 1 above)
Comment (achafetz) [reply data-raw/example_plots.R:117]: sounds pretty ideal to me :)

File: data-raw/example_plots.R (hunk 2 above)
Comment (achafetz) [reply data-raw/example_plots.R:99]: result bars plotted against target error bars 👍

**Pull request merged**